### PR TITLE
ovs_perf: Conversion Python2 to Python3

### PR DESCRIPTION
**Pull request by Nandan307**
Hi Eelco,

Kindly review the changes and get back to me.

This works in Python2 and Python3 and looks very clean too.
I have finished PVP and PP tests for L2 and L3 
in Python2 and Python3 with TRex and Xena.

Regards,
Nandan

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -2,5 +2,6 @@ Aaron Conole		aconole@redhat.com
 Eelco Chaudron		eelco@redhat.com, echaudron@gmail.com
 John Daley 		johndale@cisco.com
 Marcelo Leitner		mleitner@redhat.com
+Nandan Kulkarni		nakulkar@redhat.com, nandansanjivkulkarni@gmail.com
 Timothy Redaelli	tredaelli@redhat.com
-Vlad Buslov		tailcall@icloud.com
+Vlad Buslov		tailcall@icloud.com
```
Comment (chaudron) [inline CONTRIBUTORS]: Please re-add the new line...

File: ovs_performance.py [hunk 2]
```diff
@@ -281,6 +290,7 @@ def test_p2v2p(nr_of_flows, packet_sizes):
             start_traffic_loop_on_vm(config.dut_vm_address,
                                      config.dut_vm_nic_pci)
             warm_up_time = 0
+
```
Comment (chaudron) [inline ovs_performance.py]: Any specific reason for adding this? If not please remove the extra CR/LF

File: ovs_performance.py [hunk 3]
```diff
@@ -2155,31 +2164,32 @@ def create_multiple_graph(x, y, x_label, y_label,
...
+        gnice_y_values = dict(list(zip(list(cpu_util.keys()),
+                                       [[] for i in range(len(cpu_util))])))
+        idle_y_values = dict(list(zip(list(cpu_util.keys()),
+                                      [[] for i in range(len(cpu_util))])))
 
         y_cpu_values = [usr_y_values, nice_y_values, sys_y_values,
                         iowait_y_values, irq_y_values, soft_y_values,
                         steal_y_values, guest_y_values, gnice_y_values,
                         idle_y_values]
+
```
Comment (chaudron) [inline ovs_performance.py]: Please remove CR/LF you added

File: traffic_generator_xena.py [hunk 4]
```diff
@@ -159,12 +164,15 @@ def _configure_xena_stream(self, stream_id, traffic_flows, offset,
         else:
             raise ValueError("Unsupported traffic type for Xena tester!!!")
 
-        if (len(str(L2/L3/L4)) + 4) > packet_size:  # +4 for Ethernet CRC
-            raise ValueError("Packet size ({} bytes) to small for requested "
+        if (len(L2/L3/L4) + 4) > packet_size:  # +4 for Ethernet CRC
```
Comment (chaudron) [inline traffic_generator_xena.py:168]: This code is also in traffic_generator_trex.py so it needs fixing there also

**Review by chaudron — CHANGES_REQUESTED** [review]
Please test with XENA as its sending out carbage...

**Comment by Nandan307** [comment]
This works in Python2 and Python3 and looks very clean too.
I have finished PVP and PP tests for L2 and L3
in Python2 and Python3 with TRex and Xena.

**Review by chaudron — APPROVED** [review]

**Comment by chaudron** [comment]
Thanks, Nandan I've merged your changes. I tested almost all possible options, including DPDK and kernel datapath. Guess we need to start thinking about automation.

**Pull request merged**